packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -15,6 +15,7 @@ export * from './startPSM.js';
  * @property {string} [proposedName]
  * @property {string} keyword
  * @property {string} oracleBrand
+ * @property {number} [initialPricePct]
```

Comment:
since it's a number, just pass a float. easier to read and no loss of precision when we're talking two decimal places